cmd/nfd-master/main.go

```diff
@@ -85,14 +85,15 @@ func main() {
 func initFlags(flagset *flag.FlagSet) *master.Args {
 	args := &master.Args{
 		LabelWhiteList: utils.RegexpVal{Regexp: *regexp.MustCompile("")},
+		DenyLabelNs:    map[string]struct{}{"kubernetes.io": {}},
 	}
 
 	flagset.StringVar(&args.CaFile, "ca-file", "",
 		"Root certificate for verifying connections")
 	flagset.StringVar(&args.CertFile, "cert-file", "",
 		"Certificate used for authenticating connections")
-	flagset.Var(&args.ExtraLabelNs, "extra-label-ns",
```

Comment:
@marquiz should we have both flags at least for a release, having `ExtraLabelNS` to print a deprecation warning?  I feel that removing the flag all together could break some users deployments if they try a roll-update and the pod get's called with old flags. 
WDYT?

Reply:
@marquiz suggested that we keep the `extra-label-ns` flag because that would give flexibility to our users.

Reply:
@ArangoGutierrez yes, I don't see any reason removing the existing flag. It will also give flexibility in enabling scenarios like "deny all except for example.com/ and acme.io/"